wsi/drm/drmdisplaymanager.cpp

```diff
@@ -396,14 +396,17 @@ void DrmDisplayManager::ForceRefresh() {
   spin_lock_.unlock();
 }
 
-void DrmDisplayManager::HandleLazyInitialization() {
+bool DrmDisplayManager::HandleLazyInitialization() {
+  bool dispatched = false;
   spin_lock_.lock();
   if (release_lock_ && hwc_lock_.get()) {
```

Comment:
Do you know why release_lock_ is not true here. We shouldn't even be here if it's not true.

Reply:
release_lock_ is initialized to be false... So HWCLock is always on

Reply:
Sequence is this:
1) DisplayQueue should not update anything till lock is held.
2) Once lock is released and we get notification from HWClock thread we call Refresh.
3)In Refresh, ignore update flag is removed.
4)next update DisplayQueue calls HandleLazyInitialization

Reply:
https://github.com/intel/IA-Hardware-Composer/commit/bbba01b3d30dc51272143ee826457cbb0108960b can u check if this resolves the issue being seen. Atleast this makes it explicit.